docs/sources/flow/reference/cli/_index.md

```diff
@@ -14,10 +14,14 @@ starts Grafana Agent Flow.
 
 Available commands:
 
-* [`grafana-agent run`][run]: Start Grafana Agent Flow, given a config file.
-* [`grafana-agent fmt`][fmt]: Format a Grafana Agent Flow config file.
-* `grafana-agent completion`: Generate shell completion for the `grafana-agent` CLI.
-* `grafana-agent help`: Print help for supported commands.
+* [`grafana-agent-flow convert`][convert]: Convert a Grafana Agent configuration file.
+* [`grafana-agent-flow fmt`][fmt]: Format a Grafana Agent Flow configuration file.
+* [`grafana-agent-flow run`][run]: Start Grafana Agent Flow, given a configuration file.
+* [`grafana-agent-flow tools`][tools]: Read the WAL and provide statistical information.
+* `grafana-agent-flow completion`: Generate shell completion for the `grafana-agent-flow` CLI.
+* `grafana-agent-flow help`: Print help for supported commands.
```

Comment:
Would it be less confusing if we dropped the binary name for now?

It might remove confusion where people are unsure if the non `grafana-agent-flow` binary has these commands. 

```suggestion
* [`convert`][convert]: Convert a Grafana Agent configuration file.
* [`fmt`][fmt]: Format a Grafana Agent Flow configuration file.
* [`run`][run]: Start Grafana Agent Flow, given a configuration file.
* [`tools`][tools]: Read the WAL and provide statistical information.
* `completion`: Generate shell completion for the `grafana-agent-flow` CLI.
* `help`: Print help for supported commands.
```

Reply:
I was thinking the same while I was working on this. The binary name is not what we are really focusing on here... it's the command itself.